src-docs/src/components/guide_page/guide_page_header.tsx

```diff
@@ -128,12 +103,7 @@ export const GuidePageHeader: React.FunctionComponent<GuidePageHeaderProps> = ({
         button={button}
         isOpen={mobilePopoverIsOpen}
         closePopover={() => setMobilePopoverIsOpen(false)}>
-        <div className="guideOptionsPopover">
-          {renderGithub()}
-          <GuideSketchLink />
-          <GuideFigmaLink />
-          {renderCodeSandbox()}
```

Comment:
Are these components used anywhere else? If not, can the components themselves be deleted?